src/framework/uicomponents/view/popupview.h

```diff
@@ -100,22 +100,23 @@ class PopupView : public QObject, public QQmlParserStatus, public Injectable, pu
     explicit PopupView(QQuickItem* parent = nullptr);
     ~PopupView() override;
 
-    enum OpenPolicy {
+    enum class OpenPolicy {
         Default = 0x00000000,
         NoActivateFocus = 0x00000001,
         OpenOnContentReady = 0x00000002
     };
     Q_DECLARE_FLAGS(OpenPolicies, OpenPolicy)
     Q_FLAG(OpenPolicies)
 
-    enum ClosePolicy {
+    enum class ClosePolicy {
         NoAutoClose = 0x00000000,
         CloseOnPressOutsideParent = 0x00000001,
     };
     Q_DECLARE_FLAGS(ClosePolicies, ClosePolicy)
     Q_FLAG(ClosePolicies)
 
-    enum Placement {
+    enum class Placement {
+        Default,
         Below,
         Above

```

Comment:
maybe better to name it Placement::PreferBelow Placement::PreferAbove

Reply:
Agree, that makes more sense